docs/rules/line-comment-position.md

```diff
@@ -0,0 +1,100 @@
+# enforce position of line comments (line-comment-position)
+
+Line comments can be positioned above or beside code. This rule helps teams maintain a consistent style.
```

Comment:
This section is supposed to be an introduction to the concept that the rule covers. It helps to show some examples and explain that some style guides prefer one or the other.